.eslintrc.js

```diff
@@ -6,8 +6,7 @@ module.exports = {
   },
   extends: [
     'plugin:@typescript-eslint/recommended',
-    'prettier/@typescript-eslint',
-    'plugin:prettier/recommended'
+    'prettier'
```

Comment:
I needed to update our eslint/prettier setup as I was getting some conflicting formatting issues between the two. It's a good time to do this as this is the approach that prettier recommends and follows the design principle of separation of concerns.